docs/reference/modules/attachment.md

```diff
@@ -314,6 +324,80 @@ Usage
 node app @apostrophecms/attachment:rescale
 ```
 
+## Selected `@apostrophecms/uploadfs` module settings<a name="selecteduploadfssettings"></a>
+
+These options should be added to the `modules/@apostrophecms/uploadfs/index.js` file inside the `uploadfs` object. See the [`@apostrophecms/uploadfs` reference page](/reference/modules/uploadfs.html) for the remainder of the module options and settings.
```

Comment:
"inside the `uploadfs` option." So they know to nest under `options`.

Reply:
Changed.